### PR TITLE
Show number of loop times when an error occurs during loop.

### LINKED ISSUE #178: Explicitly state the number of loops at the point of error within the loop syntax
Display the number of loop times when an error occurs during loop

### PR DESCRIPTION
**Pull request by k1LoW**
Fix #178

### CODE REVIEW

File: operator.go [hunk 1]
```diff
@@ -656,23 +656,27 @@ func (o *operator) runInternal(ctx context.Context) error {
 					jj := j
 					o.store.loopIndex = &jj
 					if err := stepFn(o.thisT); err != nil {
+						lc := *o.store.loopIndex
 						o.store.loopIndex = nil
-						return err
+						return fmt.Errorf("loop failed (count: %d/%d): %w", lc, c, err)
```
Comment (k1LoW) [inline operator.go:661]: @k2tzumi How about an error message like this?

Comment (k2tzumi) [reply operator.go:661]: I think it's very good.
Let me verify the contents of the loop counter display.

**Comment by k2tzumi** [comment]
#  Success

```yml
vars:
  tests:
    - 1
    - 2
    - 3
    - 4
  wants:
    - 1
    - 2
    - 3
    - 4
steps:
  -
    loop:
      count: len(vars.tests)
    test: vars.tests[i] == vars.wants[i]
```

# Failure

```yml
vars:
  tests:
    - 1
    - 2
    - 3
    - 4
  wants:
    - 0
    - 2
    - 3
    - 4
steps:
  -
    loop:
      count: len(vars.tests)
    test: vars.tests[i] == vars.wants[i]
```

```log
[No Description] ... failed to run testdata/book/loop.yml: loop failed (count: 0/4): test failed on '[No Description]'.steps[0]: (vars.tests[i] == vars.wants[i]) is not true
vars.tests[i] == vars.wants[i]
├──  => ?
└──  => ?


1 scenario, 0 skipped, 1 failure
```

```yml
vars:
  tests:
    - 1
    - 2
    - 3
    - 4
  wants:
    - 1
    - 2
    - 3
    - 0
steps:
  -
    loop:
      count: len(vars.tests)
    test: vars.tests[i] == vars.wants[i]
```

```log
No Description] ... failed to run testdata/book/loop.yml: loop failed (count: 3/4): test failed on '[No Description]'.steps[0]: (vars.tests[i] == vars.wants[i]) is not true
vars.tests[i] == vars.wants[i]
├──  => ?
└──  => ?


1 scenario, 0 skipped, 1 failure
```

I felt uncomfortable because the loop counter `i` starts from 0

**Comment by k1LoW** [comment]
@k2tzumi Thank you for your feedback!!!!

> I felt uncomfortable because the loop counter i starts from 0

I thought the way the error message was presented was not good.

`i` has to be recognized as a loop INDEX.

This is because we want to use the same counting system as for `steps:`

But the error message is written in such a way that it is recognized as a loop COUNTER.

I'll try to fix the error message.

**Comment by k1LoW** [comment]
Fixed

```yaml
vars:
  tests:
    - 1
    - 2
    - 3
    - 4
  wants:
    - 1
    - 2
    - 3
    - 0
steps:
  -
    loop:
      count: len(vars.tests)
    test: vars.tests[i] == vars.wants[i]
```

```
[No Description] ... failed to run lf.yml: loop failed: test failed on '[No Description]'.steps[0].loop[3]: (vars.tests[i] == vars.wants[i]) is not true
vars.tests[i] == vars.wants[i]
├──  => ?
└──  => ?


1 scenario, 0 skipped, 1 failure
```

**Review by k2tzumi — APPROVED** [review]
LGTM

**Comment by k1LoW** [comment]
Thank you!

**Pull request merged**